store/articles/list.js

```diff
@@ -14,37 +14,34 @@ const success = storeHelper.createMutationFn(ARTICLE_LIST_RETRIEVED_SUCCESS)
 export const getters = {
   loading: state => state.status === 'loading',
   totalItems: state => state.totalItems,
-  articles: state => state.articles,
+  articles: state => state.articles[state.currentListCode],
```

Comment:
I don't follow this logic. Please at least provide some information, why it is changed or add comments.

Reply:
Because now, in state.articles we hold an items segregated by a category, we need to retrive items from this, specific category, according to url, that is changeable.

Reply:
Then you should done separate store, because this one is directly for bare articles